src/api/posts/src/processHTML/dom.js

```diff
@@ -0,0 +1,18 @@
+const jsdom = require('jsdom');
```

Comment:
We won't do any of the HTML parsing/processing of feeds in this service, you can remove this dir.

Reply:
Removing it right now will prevent my service from running at all until this process is merged. I'd either need to wait until the service is merged or just merge it there and have the person writing that service remove the directory and modify this service to use theirs